ortools_vrp.proto

```diff
@@ -43,6 +43,8 @@ message Capacity {
   float limit               = 1;
   float overload_multiplier = 2;
   bool counting             = 3;
+  float initial             = 4;
```

Comment:
I suggest `initial_limit` to make the parallel with `limit` parameter. 
```suggestion
  float initial_limit       = 4;
```


limit goes into capacity
initial_limit goes into initial_capacity